### PR TITLE
Fix handling of skipped modules with pytest.skip at module level

### PR DESCRIPTION
**Pull request by nicoddemus**
Fix #332

### CODE REVIEW

File: testing/acceptance_test.py [hunk 1]
```diff
@@ -713,9 +713,9 @@ def skip_if_unsupported_pytest_version(self):
         """Skip tests of this class if we are running in a pytest version which does not
         support warnings yet.
         """
-        from pkg_resources import parse_version
```
Comment (RonnyPfannschmidt) [inline testing/acceptance_test.py]: why this change?

**Comment by nicoddemus** [comment]
> i agree with the overall change, im confused why pkg_resources was swapped out (distutils shouldn't be used)

I thought it didn't matter much, changed to use distutils just because it is on the stdlib. Want me to swap back to `pkg_resources`?

Btw, how do you feel about dropping old pytest versions? I'm thinking requiring >=3.7

**Comment by RonnyPfannschmidt** [comment]
im entirely fine about dropping older versions - the features didnt shift much
